spec/features/cases/offender_sar_complaint/case_progressing_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+feature 'Offender SAR Case creation by a manager' do
+  given(:manager)         { find_or_create :branston_user }
+  given(:managing_team)   { create :managing_team, managers: [manager] }
+  given(:offender_sar_complaint) { create(:offender_sar_complaint).decorate }
+
+  background do
+    find_or_create :team_branston
+    login_as manager
+    cases_page.load
+    CaseClosure::MetadataSeeder.seed!
+  end
+
+  after do
+    CaseClosure::MetadataSeeder.unseed!
+  end
+
+  scenario 'progressing an offender sar case', js: true do
```

Comment:
Case -> complaint or complaint case :)

Reply:
I think for our internal terminology in the app it's fine to call complaint cases "cases", it's what they literally are. So I wouldn't change nouns around except where it's necessary for disambiguation. In the UI, it's a fair question too and even there I'd argue for sticking with "case" personally